<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string>Invalid Tokens List!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="parseButton">
        <property name="text">
         <string>Parse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="tokensListLabel">
        <property name="text">
         <string>Tokens List</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="tokensListEdit">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="syntaxTreeLabel">
        <property name="text">
         <string>Syntax Tree</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/new-file-icon.png</normaloff>:/icons/icons/new-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/exit.ico</normaloff>:/icons/icons/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
